packages/ckeditor5-language/LICENSE.md

```diff
@@ -0,0 +1,17 @@
+Software License Agreement
+==========================
+
+**CKEditor 5 Language Feature** – https://github.com/ckeditor/ckeditor5-language <br>
```

Comment:
```suggestion
**CKEditor 5 Language feature** – https://github.com/ckeditor/ckeditor5-language <br>
```

Reply:
I know we're not consistent here, but the expected format is "Camel notcase" (however it's called).